tools/validators/instance_validator/tests/fake_instances/GOOD/translation_missing_fields_on_gateway.yaml

```diff
@@ -1,12 +1,28 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Two test entities with identical points.
+
 bldg-guid:
   type: FACILITIES/BUILDING
-  code: TEST-BLDG
+  code: US-SEA-BLDG1
 
 gateway-guid:
   type: GATEWAYS/PASSTHROUGH
   connections:
     bldg-guid: CONTAINS
-  cloud_device_id: 12345
+  cloud_device_id: "2619178366980754"
```

Comment:
change cloud_device_id to something more generic like 1234567890123456

Reply:
Done.